README.md

```diff
@@ -4,12 +4,22 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/fuaftu9mvp0y488j/branch/master?svg=true)](https://ci.appveyor.com/project/cyrush/blt/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/llnl-blt/badge/?version=latest)](https://llnl-blt.readthedocs.io/en/latest/?badge=latest)
 
-A streamlined CMake build system foundation for developing HPC software
+BLT is a streamlined [CMake](https://cmake.org)-based foundation for 
```

Comment:
How about something like this:

> BLT is a [CMake](https://cmake.org)-based build system framework designed to streamline the development and simplify the maintenance of build systems for large-scale HPC applications.

or

> BLT is a [CMake](https://cmake.org)-based build system framework designed to streamline <b>B</b>uilding, <b>L</b>inking and <b>T</b>esting large-scale HPC applications.

Reply:
Thanks for the suggestions @gzagaris -- I'll update them later today.

Reply:
Thanks!

Reply:
Did you intend to make this change?

Reply:
Thanks @gzagaris -- I made the change